src/PowerShellEditorServices/Services/DebugAdapter/Handlers/LaunchAndAttachHandler.cs

```diff
@@ -395,38 +323,27 @@ void RunspaceChangedHandler(object s, RunspaceChangedEventArgs _)
                         _debugStateService.RunspaceId = id;
                         break;
 
-                        // TODO: If we don't end up setting this, we should throw
+                        // TODO: If we don't end up setting this, we should throw!
```

Comment:
If the IEnumerable is empty, throw an RpcException because we couldn't get the ID of the named runspace.